addons/crm/crm.py

```diff
@@ -27,11 +27,9 @@ class crm_stage(osv.Model):
         'sequence': fields.integer('Sequence', help="Used to order stages. Lower is better."),
         'probability': fields.float('Probability (%)', required=True, help="This percentage depicts the default/average probability of the Case for this stage to be a success"),
         'on_change': fields.boolean('Change Probability Automatically', help="Setting this stage will change the probability automatically on the opportunity."),
-        'requirements': fields.text('Requirements'),
+        'requirements': fields.text('Requirements', help="Enter here the internal requirements for this stage (ex: Offer sent to customer)"),
```

Comment:
A part of the string is missing. It should be : "Enter here the internal requirements for this stage (ex: Offer sent to customer). It will appear as a tooltip over the stage's name."